frontend/src/pages/AccessRequestPage.tsx

```diff
@@ -0,0 +1,228 @@
+
+  return (
+    <div>
+      <div>
+        <h3>Access Request</h3>
+      </div>
+      <hr />
+      <Container fluid={true}>
+        <Row className="justify-content-md-center">
+          <Formik
+            enableReinitialize={true}
+            initialValues={initialValues}
+            validationSchema={AccessRequestSchema}
+            onSubmit={(values, { setSubmitting }) => {
+              dispatch(getSubmitAccessRequestAction(toAccessRequest(values)));
+              setSubmitting(false);
+            }}
+          >
+            {props => (
+              <Form className="userInfo">
+                {(!request?.isFetching && request?.status === 201) || request?.status === 200 ? (
+                  <div className="success">
+                    <p>Your request has been submitted.</p>
+                  </div>
+                ) : null}
+
+                {initialValues.id !== 0 ? (
+                  <div className="info">
+                    <p>You will receive an email when your request is reviewed.</p>
+                  </div>
+                ) : null}
+
+                <Input
+                  label="IDIR/BCeID"
+                  field="user.username"
+                  placeholder={initialValues.user.username}
+                  readOnly={true}
+                  type="text"
+                />
+
+                <Row>
+                  <Col>
+                    <Input
+                      label="First Name"
+                      field="user.firstName"
+                      placeholder={initialValues.user.firstName}
+                      readOnly={true}
+                      type="text"
+                    />
+                  </Col>
+                  <Col>
+                    <Input
+                      label="Last Name"
+                      field="user.lastName"
+                      placeholder={initialValues.user.lastName}
+                      readOnly={true}
+                      type="text"
+                    />
+                  </Col>
+                </Row>
+
+                <Input
+                  label="Email"
+                  field="user.email"
+                  placeholder={initialValues.user.email}
+                  readOnly={true}
+                  type="email"
+                />
+
+                {checkAgencies}
+
+                <Input
+                  label="Position"
+                  field="user.position"
+                  placeholder="e.g) Director, Real Estate and Stakeholder Engagement"
+                  type="text"
+                />
+
+                {checkRoles}
+
+                <TextArea
+                  label="Notes"
+                  field="note"
+                  placeholder="Please specify why you need access to this site."
+                />
+
+                <p>
+                  By clicking request, you agree to our{' '}
+                  <a href="https://www2.gov.bc.ca//gov/content/home/disclaimer">
```

Comment:
Might be good to store these urls in case they are reused, such as in strings.tsx

Reply:
done